src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/CancelablePromise.cs

```diff
@@ -27,7 +27,9 @@ public static unsafe void CancelPromise(Task promise)
             Interop.Runtime.CancelPromise(holder.GCHandle);
 #else
 
```

Comment:
This `#if` check seems redundant since we're inside the `#else` block for the previous `#if !FEATURE_WASM_MANAGED_THREADS`  (e.g. the thing we're testing is already known).

Was something else intended? If not, we can delete line 30 and 32, because they are _known_ to be active here

Full conditional context [shown here](https://github.com/dotnet/runtime/pull/113046/files#diff-6651067349f868184cc9fa01dfa74d6dce35a3a5f4f3dc7589adc2c03328a23eR22-R53)

Reply:
I missed that one

Reply:
Created a PR for this https://github.com/dotnet/runtime/pull/113284